File: ios/ConfCamera.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(ConfCamera, NSObject)

RCT_EXTERN_METHOD(openCamera:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)
@end

File: ios/ConfCamera.swift
@objc(ConfCamera)
class ConfCamera: NSObject {

    @objc(openCamera:withRejecter:)
    func openCamera(resolve:RCTPromiseResolveBlock.reject:RCTPromiseResolveBlock) -> Void {
      let imagePicker = UIImagePickerController();
      imagePicker.sourceType = .photoLibrary;

      DispatchQueue.main.async {
        RCTPresentedViewController()?.present(imagePicker, animated: true)
      }

      resolve("Camera opened success");
    }

}
